src/wp-admin/menu.php

```diff
@@ -210,7 +210,7 @@
 	$submenu['themes.php'][6] = array(
 		__( 'Template Parts' ),
 		'edit_theme_options',
-		'site-editor.php?postType=wp_template_part',
+		'site-editor.php?postType=wp_template_part&path=/wp_template_part/all',
```

Comment:
If I remember correctly, there's no need to escape static strings.

Reply:
No need to escape this string as it is not exposed to user edits, indeed.